datamodel/testdata.d/08_01_test_data_ebl.sql

```diff
@@ -496,20 +512,14 @@ INSERT INTO dcsa_im_v3_0.location (
 
 INSERT INTO dcsa_im_v3_0.shipping_instruction (
     id,
-    transport_document_type,
     is_shipped_onboard_type,
     number_of_copies,
-    number_of_originals,
```

Comment:
Please re-insert number_of_originals and call it **requested_number_of_originals**
(I cannot make a suggestion to deleted code...

Reply:
Done